core/string/ustring.cpp

```diff
@@ -4788,6 +4799,17 @@ String TTR(const String &p_text, const String &p_context) {
 	return p_text;
 }
 
+/**
+ * "Tools TRanslate for N items". Performs string replacement for
+ * internationalization within the editor. A translation context can optionally
+ * be specified to disambiguate between identical source strings in
+ * translations. Use `TTR()` if the string doesn't need dynamic plural form.
+ * When placeholders are desired, use
+ * `vformat(TTRN("%d item", "%d items", some_integer), some_integer)`.
```

Comment:
I am not 100% sure this is the correct way to do pluralized translation with a placeholder. Can anyone check?

Reply:
It's similar to how `ngettext` is used: https://www.gnu.org/software/gettext/manual/html_node/Plural-forms.html

But unlike `printf`, `vformat` will complain about excess arguments and return empty string. So something like this won't work:

```c++
vformat(TTRN("One item", "%d items", some_integer), some_integer)
```

Reply:
Done, I added a mention about the requirement to use the placeholder in both strings.